### ISSUE TITLE
Feedback

### ISSUE THREAD

**Issue by IgorTodorovskiIBM** [DevonianTeuchter/meta#1]
1. Can we improve this error message `Internal Error. $ZOPEN_CA must be set` to recommend to the user to run `zopen init`

2. There is an extra newline at the top after invoking: `zopen init`

3. If the bootstrap rexx script fails, can you print the error message. 

4. We got this error (after adding a print statement for $bootstrap)
```
Generating HWT kdb... Generating HWT stash... Web toolkit function failed: hwthconn:0;RetCode:262(0x106 );SVC:;RSN:;DSC:
```

5. Instead of using the bootstrap rexx script, can we embed a stable version of curl and use that to download the latest version of curl and/or check if the curl in the user's PATH is a valid curl and potentially skip downloading curl.

6. We got this message after rerunning `zopen-init`:
ln: FSUM6245 link to target "$HOME/zopen-test/usr/share/zopen/boot" failed: EDC5117I File exists.
ln: FSUM6245 link to target "$HOME/zopen-test/usr/share/zopen/prod" failed: EDC5117I File exists.

7. A couple of help suggestions:
* Use z/OS Open Tools vs zOpen Tool
* This is incomplete: `queries zOpen Tool packages (see`

**Comment by DevonianTeuchter** [comment]
1/2. Resolved in 0.0.2/stable
3/4/5. Bootstrap.rexx is deprecated in 0.0.2/stable as there is now a version of curl shipped directly; the REXX script is only there as a fallback and will be removed at a later date
6. 0.0.2/stable should handle reinitializing better (with a --re-init parameter and existence checking
7. Fixed messages and names

The new release 0.0.2 and/or code in the stable branch should handle error scenarios better

**Comment by IgorTodorovskiIBM** [comment]
This feedback is now outdated, closing
